src/components/textWithIcon/TextWithIcon.tsx

```diff
@@ -0,0 +1,26 @@
+import { View, Text } from 'react-native';
+import MaterialCommunityIcons from 'react-native-vector-icons/MaterialCommunityIcons';
+
+import colors from '@constants/colors';
```

Comment:
move this `color` to theme, why is this not resolved yet

Reply:
I will refactor this when I implement the `theme` feature